src/analyze/analyze-security.h

```diff
@@ -3,9 +3,10 @@
 
 #include "sd-bus.h"
 
-typedef enum AnalyzeSecurityFlags {
-        ANALYZE_SECURITY_SHORT             = 1 << 0,
-        ANALYZE_SECURITY_ONLY_LOADED       = 1 << 1,
+typedef enum AnalyzeSecurityFlags
+{
+        ANALYZE_SECURITY_SHORT = 1 << 0,
+        ANALYZE_SECURITY_ONLY_LOADED = 1 << 1,
         ANALYZE_SECURITY_ONLY_LONG_RUNNING = 1 << 2,
 } AnalyzeSecurityFlags;
```

Comment:
i would prefer if enum value definitions would be aligned in a table, if they are specified, like the old code did it

Reply:
There is an option to align the assignment, but this is really undesirable to apply to the whole code base. These cases could again be custom-aligned with comments, if required:
```cpp
typedef enum AnalyzeSecurityFlags
{
        ANALYZE_SECURITY_SHORT /*********/ = 1 << 0,
        ANALYZE_SECURITY_ONLY_LOADED /***/ = 1 << 1,
        ANALYZE_SECURITY_ONLY_LONG_RUNNING = 1 << 2,
} AnalyzeSecurityFlags;
```